extensions/blocks/contact-form/components/jetpack-newsletter-integration-settings.js

```diff
@@ -44,6 +55,190 @@ const NoConsentBlockSettings = () => {
 	);
 };
 
+const pluginStateEnum = Object.freeze( {
+	ACTIVE: 1,
+	INSTALLED: 2,
+	NOT_INSTALLED: 3,
+} );
+
+const CreativeMailPluginErrorState = ( { error } ) => {
+	return (
+		<p>
+			<em style={ { color: 'red' } }>
```

Comment:
I would recommend using the `Notice` component to display those notices, for a more consistent behaviour with the rest of the editor notices.

You can check `extensions/blocks/instagram-gallery/edit.js` for an example of how this can be done.

Reply:
![image](https://user-images.githubusercontent.com/47211808/91051371-4fc6ec00-e620-11ea-8b76-278f71cf7ed0.png)
done